src/runtime/serialization/Serializer.cpp

```diff
@@ -65,7 +73,7 @@ bool Serializer::serializeInto(const Value& value, std::ostringstream& output)
     return false;
 }
 
-std::unique_ptr<SerializedValue> Serializer::deserializeFrom(std::istringstream& input)
+std::unique_ptr<SerializedValue> Serializer::deserializeFrom(Context* context, std::istringstream& input)
```

Comment:
Parameter `Context* context` seems nowhere used now.

Reply:
You are right!
Good point